node/value.go

```diff
@@ -74,6 +75,24 @@ func NewValue(typ *meta.Type, v interface{}) (val.Value, error) {
 	case val.FmtUnion:
 		cvt, _, err := val.ConvOneOf(typ.UnionFormats(), v)
 		return cvt, err
+	case val.FmtUnionList:
+		if v == nil {
+			return nil, nil
+		}
+		sliceValue := reflect.ValueOf(v)
```

Comment:
can you move this code to toUnionList instead of making this inline to keep consistent w/other conversion handlers?